truncate/table_schema.go

```diff
@@ -54,7 +63,17 @@ type indexSchema struct {
 	parentTableName string
 }
 
-func fetchTableSchemas(ctx context.Context, client *spanner.Client, targetTables, excludeTables []string) ([]*tableSchema, error) {
+// tableLineage represents a table schema and its ancestors and descendants.
+// This is used to represent inter-table relationships.
+// The order of ancestors and descendants are not guaranteed.
+type tableLineage struct {
+	tableSchema *tableSchema
+	ancestors   []*tableSchema
+	descendants []*tableSchema
+}
```

Comment:
Introduced a new struct to separate the logic and simplify for managing table relationships, enhancing code readability and maintainability.

And we can use descendants to fix #18 .